packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/ios/integrationsteps.md

```diff
@@ -1,5 +1,181 @@
+## Integration steps
+
+### Summary
+
+There are many different ways to integrate the Swift SDK into your app. Using
+the sample code as a wrapper around the SDK, this guide describes the [steps](#steps)
+involved in integrating this use case into your application. Feel free to modify
+the sample code to fit your individual needs. The diagram below illustrates
+how the sample code and SDK fit into the overall integration of your app.
+
+<div class="common-image-format">
+
+ ![Diagram showing the integration flow of the sample app and Swift SDK](/img/oie-embedded-sdk/oie-embedded-sdk-swift-sample-code-overview.png
+ "Swift sample and sdk diagram")
+
+</div>
+
+### Steps
+
+#### Step 1: Initialize SocialLogin
+
+The first step is to initialize `SocialLogin` and pass in a
```

Comment:
How about "The first step is to initialize `SocialLogin`, passing in a `configuration` object as a parameter." How it is written currently sounds like passing in the param is a separate step.

Reply:
Sentence has been reworded.

Reply:
Wording is looking good; just one small nitpick — need an exra space between "a" and "`configuration`"

Reply:
a and 'configuration' have been separated.

Reply:
👍